plugin-dev/Source/Sentry/Private/Apple/SentrySubsystemApple.cpp

```diff
@@ -79,7 +79,6 @@ void SentrySubsystemApple::InitWithSettings(const USentrySettings* settings, USe
 				[options addInAppExclude:it->GetNSString()];
 			}
 			options.enableAppHangTracking = settings->EnableAppNotRespondingTracking;
-			options.enableTracing = settings->EnableTracing;
```

Comment:
Why was that removed?

Reply:
`enableTracing` was deprecated and now providing one of the `sampleRate`/`samplingFunction` options explicitly is sufficient to let the SDK know that the tracing is enabled.